platforms/enterprise/enterprise-operations/src/main/java/org/gradle/internal/configurationcache/ConfigurationCacheCheckFingerprintBuildOperationType.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.configurationcache;
```

Comment:
🤔 Should all the new CC operations go into the package in a "new" build operations package structure?

We've introduced `org.gradle.operations.configuration` for `IsolatedProjectsSettingsFinalizedProgressDetails`.

I understand it's a "+1 standard" problem, but we have to start somewhere.

Cc @wolfs, should we add an arch test that checks if the new build operations are not added under `org.gradle.operations` ?

Reply:
> I think it would make sense to add it to `org.gradle.operations.configuration`.

Done